easybuild/easyconfigs/m/mrFAST/mrFAST-2.6.0.1-goolf-1.4.10.eb

```diff
@@ -0,0 +1,29 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Pablo Escobar Lopez
+# sciCORE - University of Basel
+# SIB Swiss Institute of Bioinformatics 
+
+easyblock = 'MakeCp'
+
+name = 'mrFAST'
+version = '2.6.0.1'
+
+homepage = 'http://mrfast.sourceforge.net/'
+description = """ mrFAST & mrsFAST are designed to map short reads generated with the 
+ Illumina platform to reference genome assemblies; in a fast and memory-efficient manner. """
+
+toolchain = {'name': 'goolf', 'version': '1.4.10'}
+
+source_urls = [SOURCEFORGE_SOURCE]
+sources = [SOURCELOWER_TAR_GZ]
+
+parallel = 1
+
```

Comment:
@pescobar please include the `buildopts = ' CC="$CC"'`, so this does what it's supposed to do when `--try-toolchain` is used